spec/lib/onebox/engine/github_actions_onebox_spec.rb

```diff
@@ -4,16 +4,18 @@
   describe "PR check run" do
     before do
       @link = "https://github.com/discourse/discourse/pull/13128/checks?check_run_id=2660861130"
+      @pr_run_uri = "https://api.github.com/repos/discourse/discourse/pulls/13128"
```

Comment:
We could use `let` instead of instance variables here. I guess those are old specs for being written that way?

Reply:
Yeah they are weird old specs with a bit of a strange setup, would prefer not to go too deep in this rabbit hole atm :)

Reply:
No worries! I may do a quick followup :wink:

Reply:
Sure, LMK if you want me to review at that point!

Reply:
Done in https://github.com/discourse/discourse/pull/27871, if you want to have a look :slightly_smiling_face: (Jarek already approved the PR :sweat_smile:)